Node.gitignore

```diff
@@ -108,9 +108,14 @@ dist
 # Stores VSCode versions used for testing VSCode extensions
 .vscode-test
 
-# yarn v2
-.yarn/cache
-.yarn/unplugged
-.yarn/build-state.yml
-.yarn/install-state.gz
+# Yarn modern
+# Comment for zero-install
 .pnp.*
+.yarn/*
+# Uncomment for zero-install
+# !.yarn/cache
+!.yarn/patches
+!.yarn/plugins
+!.yarn/releases
```

Comment:
Does it really make sense to add those files:
```
ll .yarn/releases/yarn-3.2.4.cjs
-rwxr-xr-x 1 christof christof 2.1M Okt 19 21:43 .yarn/releases/yarn-3.2.4.cjs
```

Reply:
This is what's currently recommended in the docs. However, Yarn 4 seems like it will have better support for managing versions with corepack, so this may change in the future.

Reply:
Also you can use `.gitattributes` if you don't want these files showing up in diffs.